contrib/vscode/build.sh

```diff
@@ -8,15 +8,15 @@ if [ -z "$ROOT" ]; then
 fi
 
 # build in tree
-rm -rf build ${ROOT}/dist
+sudo rm -rf build ${ROOT}/dist
```

Comment:
can you please prefix the commit message for this file with `trivial:`

Reply:
ok, in next patch.

Reply:
Done.